mathics/core/atoms.py

```diff
@@ -640,14 +670,26 @@ def __new__(cls, real, imag):
         if isinstance(imag, MachineReal) and not isinstance(real, MachineReal):
             real = real.round()
 
-        self.real = real
-        self.imag = imag
+        value = (real, imag)
```

Comment:
Why not a class first element here?

Reply:
That is a possibility.  It might also be possible to use a python complex for most of the cases where that works. There may be edge cases where it doesn't and then some other representation could be used in those special cases. 

Right now though, since this is new we aren't using this. So rather than over-engineer, this is something simple and provisional. As the need arises, it can change. But let's do this based on need. 

The reason to add a value was that this is the way all of the other numeric constants work.